bitnami/etcd/3.3/debian-11/rootfs/opt/bitnami/scripts/libetcd.sh

```diff
@@ -661,7 +661,7 @@ etcd_initialize() {
             if ! is_healthy_etcd_cluster; then
                 warn "Cluster not responding!"
                 if is_boolean_yes "$ETCD_DISASTER_RECOVERY"; then
-                    latest_snapshot_file="$(find /snapshots/ -maxdepth 1 -type f -name 'db-*' | sort | tail -n 1)"
+                    latest_snapshot_file="$(find ${ETCD_SNAPSHOTS_DIR}/ -maxdepth 1 -type f -name 'db-*' | sort | tail -n 1)"
```

Comment:
Please double quote to prevent linter errors.
```suggestion
                    latest_snapshot_file="$(find "${ETCD_SNAPSHOTS_DIR}/" -maxdepth 1 -type f -name 'db-*' | sort | tail -n 1)"
```

Reply:
Thanks for the comment!  I've pushed the change to all three debian versions..